pgbouncer/Makefile

```diff
@@ -0,0 +1,32 @@
+VM_NAME := pgbouncer-sampleapp
```

Comment:
Since the repo name uses `integration-sample-apps`, can you make it `pgbouncer-sample-app` and update name usage to follow the flow